### DISCUSSION TITLE
AG Flashing - starts and quits continuously

### DISCUSSION THREAD

**Discussion by Umpalumpe** [apohl79/audiogridder#494] [Help]
So I got a problem with an Ableton session with a few instances of Audiogridder:
It starts and quits like crazy - It's flashing and I'm not able to quit the AudiogridderServer app, since it's starts and quits so fast. So is this a known bug for v 1.1.1 and is there a way to solve it without having to delete all my AG instances in my project?

**Comment by CaBuffalo** [comment]
Ver 1.1.1 is very stable so far. From my experience, this behavior might be
triggered by certain plugins. You might to disable some of the plugins and
try to find out what causes the restarting of the server plugin.

On Sat, Apr 17, 2021 at 12:16 PM Umpalumpe ***@***.***> wrote:

> So I got a problem with an Ableton session with a few instances of
> Audiogridder:
> It starts and quits like crazy - It's flashing and I'm not able to quit
> the AudiogridderServer app, since it's starts and quits so fast. So is this
> a known bug for v 1.1.1 and is there a way to solve it without having to
> delete all my AG instances in my project?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/494>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HGEUKVM4PPUUBVQEDTJHNAJANCNFSM43DL5G2A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Umpalumpe** [comment]
Okay, might be an issue with multiple instances of DDMF Metaplugin, which is unfortunate, since I'm using it a lot...
